x/ibc/core/04-channel/types/packet_test.go

```diff
@@ -5,9 +5,39 @@ import (
 
 	"github.com/stretchr/testify/require"
 
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	clienttypes "github.com/cosmos/cosmos-sdk/x/ibc/core/02-client/types"
 	"github.com/cosmos/cosmos-sdk/x/ibc/core/04-channel/types"
 )
 
+func TestCommitPacket(t *testing.T) {
+	testCases := []struct {
+		packet  types.Packet
+		expPass bool
+		errMsg  string
+	}{
+		{types.NewPacket(validPacketData, 1, portid, chanid, cpportid, cpchanid, timeoutHeight, timeoutTimestamp), true, ""},
+		{types.NewPacket(validPacketData, 0, portid, chanid, cpportid, cpchanid, timeoutHeight, timeoutTimestamp), false, "invalid sequence"},
```

Comment:
this test doesn't make sense to me, `CommitPacket` doesn't use the packet sequence at all?